From df5848f423ec0f4b3e519900014349a0c00b1969 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Mon, 17 Aug 2020 16:36:03 +0200
Subject: [PATCH] Do not disable introspection in cross builds.

Yocto can and does support introspection cross builds, through
running target binaries under qemu emulation.

Upstream-Status: Pending
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 1865633..5e1787c 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -634,7 +634,7 @@ if have_gobject
   gir = find_program('g-ir-scanner', required: get_option('introspection'))
   build_gir = gir.found()
 
-  build_gir = build_gir and not meson.is_cross_build()
+  build_gir = build_gir
   if not build_gir and get_option('introspection').enabled()
     error('Introspection support is requested but it isn\'t available in cross builds')
   endif
